poppy/wfe.py

```diff
@@ -20,16 +21,72 @@
 
 __all__ = ['WavefrontError', 'ParameterizedWFE', 'ZernikeWFE']
 
+def _accept_wavefront_or_meters(f):
+    """Decorator that ensures the first positional method argument
+    is a poppy.Wavefront or a floating point number of meters
+    for a wavelength
+    """
+
```

Comment:
Can you remind me what the motivation is for allowing either Wavefront or floating point wavelength as the first argument?  I guess it's just a convenience because it's often relevant to generate Zernikes over an abstract pupil without defining a spatial scale. 

I'm mostly wondering if this decorator is only relevant here in `wfe.py` or if there are conceivably other use cases such that it should be promoted to `optics.py`